resources/src/jsMain/kotlin/dev/icerock/moko/resources/desc/ResourceFormattedStringDesc.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources.desc
+
+import dev.icerock.moko.resources.StringResource
+import dev.icerock.moko.resources.provider.JsStringProvider
+
+actual class ResourceFormattedStringDesc actual constructor(
+    val stringRes: StringResource,
+    val args: List<Any>
+) : StringDesc {
+    override suspend fun localized(): String =
+        localized(stringRes.loader.getOrLoad())
+
+    override fun localized(provider: JsStringProvider): String {
+        return stringRes.localized(
+            provider = provider,
+            locale = StringDesc.localeType.locale,
+            args = args.toTypedArray()
```

Comment:
here not missed star?

Reply:
no, since I references to this param by name. btw, everything compiles and works fine.